### DISCUSSION TITLE
Set Power factor top Hm-1500

### DISCUSSION THREAD

**Discussion by tammof** [tbnobody/OpenDTU#550] [Q&A]
Hy all,

Can i set the power factor of an HM-1500 via opendtu to provide reactive power to the grid. 
I need to set it to 0.95.

If so, can someone please point me to the relevant documentation?

Thanks a Lot!

**Comment by D3R-ST3FAN** [comment]
Grid profiles (reactive power control is part of it) can not yet be modified with OpenDTU nor Ahoy.
You will need to use the original DTU and an Hoymiles (S Miles Cloud) installer account to do so.

**Reply by tammof** [reply]
Will a dtu Lite suffice to do so?

Thanks a Million!

**Reply by D3R-ST3FAN** [reply]
> Will a dtu Lite suffice to do so?
yes

**Reply by tammof** [reply]
Thank you

**Reply by Hubert1975** [reply]
Schade, würde diese Funktion für Österreich TOR Einstellung (festen cosPHi=1 und fester Blindleistung Qfix=0) auch benötigen. kann das nicht eingebaut werden. dann werden viele Anlagen mit open DTU gebaut!

**Comment by Forhorse88** [comment]
Sorry to bring this topic up.
Is it planned in a future version of openDTU to have the possibility to adjust the power factor ?

**Reply by Hubert1975** [reply]
Das würd ich auch benötigen, hab schon alles durchgesucht. wird ja am Microwechselrichter eingestellt und über die open DTU  gibt's ja die Verbindung! das teure Teil von Hoymiles will ich nicht kaufen.
weiß jemand was voreingestellt ist??

**Comment by Solarteur** [comment]
Laut Datenblatt für einen HM-600 >0,99
![grafik](https://user-images.githubusercontent.com/117977338/233825383-211ddbb1-03a5-4ede-8bd2-2c76d244b4e1.png)

**Reply by Hubert1975** [reply]
Besten Dank für die Daten!
leider muß in Österreich fix eingestellt werden..... und das geht leider nur mir DTU vom Hersteller, ausser jemand kennt sich aus und bekommen ein Update!
